### ISSUE TITLE
Suspected overfitting

### ISSUE THREAD

**Issue by lss0726** [HighwayWu/InpaintingForensics#10]
I used gc_dresden_train as the training set and DiversePaintingDataset/GC as the validation set for training. Why is the AUC of the validation set always around 50? I suspect it is overfitting. Could you please help me answer this? Here is the log of this training session
![图片](https://github.com/HighwayWu/InpaintingForensics/assets/54062017/ed5bee6b-20ff-474c-8664-0b831d42a4dc)
